.github/workflows/lint.yml

```
@@ -13,7 +13,7 @@ jobs:
     steps:
       - uses: actions/setup-go@v4
```

actionlint (1.7.10)

[error] the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)